otherlibs/win32unix/stat.c

```diff
@@ -117,23 +117,13 @@ static value stat_aux(int use_64, __int64 st_ino, struct _stat64 *buf)
 
 static int convert_time(FILETIME* time, __time64_t* result, __time64_t def)
 {
-  SYSTEMTIME sys;
-  FILETIME local;
+  unsigned __int64 utime = *((unsigned __int64*)time);
```

Comment:
MSDN says somewhere "Do not cast a pointer to a FILETIME structure to either a ULARGE_INTEGER* or __int64* value because it can cause alignment faults on 64-bit Windows." and recommends to copy through a ULARGE_INTEGER structure instead.